2.0/PearlGoldOliveCoveredGlassDish/README.md

```diff
@@ -0,0 +1,90 @@
+# PearlGoldOliveCoveredGlassDish 
+
+## Screenshots
+
+![screenshot Babylon.js](screenshot/screenshot_large.jpg)
```

Comment:
The file is called `screenshot_Large.jpg` (with capital L) so it doesn't show up on case-sensitive filesystems, including GitHub preview. Please either rename it or fix the link.

Reply:
Should all the screenshots be renamed using lowercase? Or just screenshot_Large.jpg?

Reply:
Just `screenshot_Large.jpg`. There are many mixed-case files in the repo.